partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -789,11 +790,52 @@ parameter
 
 varRefExpr
     : qualifier=AT_SIGN? ident=(IDENTIFIER|IDENTIFIER_QUOTED)   # VariableIdentifier
-    | qualifier=AT_SIGN? key=nonReservedKeywords                # VariableKeyword
-    ;
-
-nonReservedKeywords
-    : EXCLUDED
+    | qualifier=AT_SIGN? key=nonReserved                # VariableKeyword
+    ;
+
+nonReserved
```

Comment:
(self-review) -- the [non-reserved words](https://ronsavage.github.io/SQL/sql-99.bnf.html#non-reserved%20word) were taken directly from the SQL-99 grammar. At the end are some, PartiQL specific-keywords marked as non-reserved.

If you'll notice, [postgresql](https://www.postgresql.org/docs/current/sql-keywords-appendix.html) along with other SQL dialects use their own set of non-reserved keywords that diverges from the SQL standards. Also between different SQL standards, there are some changes to the reserved vs non-reserved keywords. We choose to follow SQL-99 as we do in other places in the project.